tests/aws/services/lambda_/test_lambda.py

```diff
@@ -201,6 +202,63 @@ def test_large_payloads(self, caplog, create_lambda_function, aws_client):
         assert "FunctionError" not in result
         assert payload == json.load(result["Payload"])
 
+    @markers.aws.validated
+    def test_lambda_large_reponse(self, caplog, create_lambda_function, aws_client):
+        # Set the loglevel to INFO for this test to avoid breaking a CI environment (due to excessive log outputs)
+        caplog.set_level(logging.INFO)
+
+        function_name = f"large_response-{short_uid()}"
+        create_lambda_function(
+            handler_file=TEST_LAMBDA_CUSTOM_RESPONSE_SIZE,
+            func_name=function_name,
+            runtime=Runtime.python3_10,
```

Comment:
nit: could use 3.12

Reply:
same as alias PR, let's update them all together when implementing API support for LoggingConfig :+1: